<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JKOtherEnvrionmentDlg</class>
 <widget class="QDialog" name="JKOtherEnvrionmentDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JKOtherEnvrionmentDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="environmentLab">
         <property name="text">
          <string>输入环境：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="environmentText"/>
       </item>
       <item>
        <widget class="QCheckBox" name="isPrivateCheck">
         <property name="text">
          <string>是否为私有云</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
